web/src/views/FindSpeaker.vue

```diff
@@ -92,31 +92,28 @@ export default {
     PaginationRow,
     NoResults,
   },
-  data() {
-    return {
-      speakers: [],
-      prefectures: [],
-      topicList: [],
-      languageList: [],
-      error: null,
-      selectedSpeaker: undefined,
-      showDialog: false,
-      showSuccess: false,
-      page: 0,
-      pageSize: 50,
-      isLastPage: false,
-      isLoading: true,
-    };
-  },
+  data: () => ({
+    speakers: [],
+    prefectures: [],
+    topicList: [],
+    languageList: [],
+    error: null,
+    selectedSpeaker: undefined,
+    showDialog: false,
+    showSuccess: false,
+    page: 0,
+    pageSize: 50,
+    isLastPage: false,
+    isLoading: true,
+  }),
   computed: {
     selectedName() {
-      if (this.selectedSpeaker) {
-        if (this.$i18n.locale === 'ja') {
-          return this.selectedSpeaker.get('name_ja') || this.selectedSpeaker.get('name_en') || '';
-        }
-        return this.selectedSpeaker.get('name_en') || '';
+      if (!this.selectedSpeaker) return '';
```

Comment:
Can we keep this PR focused on search and avoid making other refactors? It gets confusing to follow the changes.